recipe/install_llvm.sh

```diff
@@ -30,13 +30,26 @@ elif [[ "${PKG_NAME}" == libllvm* ]]; then
     mv ./temp_prefix/lib/libLLVM-${MAJOR_EXT}${SHLIB_EXT} $PREFIX/lib
     mv ./temp_prefix/lib/lib*.so.${SOVER_EXT} $PREFIX/lib || true
     mv ./temp_prefix/lib/lib*.${SOVER_EXT}.dylib $PREFIX/lib || true
-elif [[ "${PKG_NAME}" == "llvm-tools" ]]; then
+elif [[ "${PKG_NAME}" == "llvm-tools-${MAJOR_EXT}" ]]; then
     cmake --install ./build --prefix=./temp_prefix
-    # everything in /bin & /share
-    mv ./temp_prefix/bin/* $PREFIX/bin
-    mv ./temp_prefix/share/* $PREFIX/share
+    # install all binaries with a -${MAJOR_EXT}
+    pushd ./temp_prefix
+      for f in bin/*; do
+        cp $f $PREFIX/bin/$(basename $f)-${MAJOR_EXT}
+      done
+    popd
     # except one binary that belongs to llvmdev
-    rm $PREFIX/bin/llvm-config
+    rm $PREFIX/bin/llvm-config-${MAJOR_EXT}
+elif [[ "${PKG_NAME}" == "llvm-tools-${MAJOR_EXT}" ]]; then
+    cmake --install ./build --prefix=./temp_prefix
+    # Install a symlink without the major version
+    pushd ./temp_prefix
+      for f in bin/*; do
+        ln -sf $PREFIX/bin/$(basename $f)-${MAJOR_EXT} $PREFIX/bin/$(basename $f)
+      done
+    popd
```

Comment:
Aren't you going to be creating a symlink to `$PREFIX/bin/llvm-config` here? AFAIU we'd need to delete `$PREFIX/bin/llvm-config-${MAJOR_EXT}` before this loop.

Reply:
No, since `llvm-config` is going to be in `llvmdev`, I'm not going to add a versioned symlink.

Reply:
My point is that `llvm-config` is going to be in `temp_prefix/bin` after installation, so it will get picked up in the for-loop

Reply:
Ah, got it. Thanks